web-apps/chat/app.py

```diff
@@ -81,7 +81,7 @@ def inference(latest_message, history):
             else:
                 if role != "assistant":
                     log.warn(f"Message role {role} converted to 'assistant'")
-                    context.append(AIMessage(content=(content or "")))
+                context.append(AIMessage(content=(content or "")))
```

Comment:
Is this all that was required to fix the reasoning models?

Reply:
I haven't checked reasoning models yet, that fix was required for anything after the second response

Reply:
Fixed reasoning models, and tested with all the site-specific overrides. All looks good